ideascube/conf/idb_fra_aurore.py

```diff
@@ -1,28 +0,0 @@
-"""Ideasbox Aurore in Paris Clichy, France"""
-from .idb_fra import *  # pragma: no flakes
-
-
-IDEASCUBE_NAME = u"Aurore"
```

Comment:
If we do this, then the server will be called « Ideas Cube » in the UI (top-left corner of the app) instead of « Aurore ».

Is that what we want?

If not, you must run `ideascube config set server site-name Aurore` on the server, to get the name set properly.